src/extlibs/hrvo/hrvo_test.cpp

```diff
@@ -0,0 +1,247 @@
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <fstream>
+
+#include "simulator.h"
```

Comment:
we should include the full path, i.e. extlibs/hrvo/simulator.h

Reply:
Made all includes to use full path.